spec/features/site/user_visits_home_page_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'User visits home page' do
   context 'when they are signed in' do
     background { sign_in_with(user.email, user.password) }
-    given(:user) { create(:user) }
+    given(:user) { create(:user, :moderator) }
```

Comment:
How are you enforcing access control, or are you not yet?

I would recommend pundit, and I would add to this branch.